scripts/setup_uv.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+
+set -e  # Exit on error
+
+#!/bin/bash
+
```

Comment:
I haven't seen something like this before: why #!/usr/bin/env bash and then #!/bin/bash?